Images/Buttons.css

```diff
@@ -0,0 +1,56 @@
+.header {
+  text-align: center;
+}
+.btn-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+.btn {
+  height: 2.3rem;
+  width: 10rem;
+  cursor: pointer;
+  border: 1px solid transparent;
+  border-radius: 4px;
+  font-weight: 600;
+}
+
+.btn-primary {
+  color: #fff;
+  /* background-color: #0d6efd; */
+  background-color: #045de9;
+  background-image: linear-gradient(315deg, #045de9 0%, #09c6f9 74%);
+  border-color: #a7c3f2;
+}
+
+.btn-Secondary {
```

Comment:
Should be btn-seconday

Reply:
Noted Humza Hasan
thanks for the comment